receiver/filelogreceiver/README.md

```diff
@@ -153,4 +153,10 @@ The above configuration will read logs from the "simple.log" file. Some examples
 2023-06-20 12:50:00 DEBUG This is a test debug message
 ```
 
+## Fault tolerance, recovery and proper offset tracking
 
+`storage` setting allows to define the proper storage extension to be used for storing file offsets. 
+File offsets allow the receiver to pick up where it left off in the case of a collector restart. 
+If no storage extension is used, the receiver will manage offsets in memory only.
```

Comment:
This kind of just restates the information in the parameters table. I think it would be more helpful to articulate what offset tracking does and doesn't do.

Specifically, we can say something like the following:
"While the `storage` parameter can ensure that log files are consumed accurately, it is possible that logs are dropped while moving downstream through other components in the collector. For additional resiliency, see (example)."